guides/v2.1/cloud/project/project-multi-sites.md

```diff
@@ -160,7 +160,7 @@ Instead of configuring an {% glossarytooltip b14ef3d8-51fd-48fe-94df-ed069afb2cd
 5.	Save your changes to `magento-vars.php` and exit the text editor.
 
 ### Deploy and test on the Integration server {#cloud-multi-stores-deploy}
-The final step is to push your changes to your Magento Entperise Cloud Edition server and test your site there.
+The final step is to push your changes to your Magento Enterprise Cloud Edition server and test your site there.
```

Comment:
No {{site.data.var.ece}} variable here?

Reply:
Actually, @keharper is correct. It should be the variable so that it shows Magento Commerce Cloud. Enterprise is sooooo yesterday. ;)